src/devtools/client/debugger/src/actions/source-actors.js

```diff
@@ -88,11 +88,14 @@ export const loadSourceActorBreakpointHitCounts = memoizeableAction(
       ]).join("-");
     },
     getValue: ({ id }, { getState }) => getSourceActorBreakpointHitCounts(getState(), id),
-    action: async ({ id }, { dispatch, getState, client }) => {
+    action: async ({ id, onFailure }, { dispatch, getState, client }) => {
       await dispatch({
```

Comment:
You probably dont need to pass `onFailure` in. Typically using `.catch` is nicer.